proto/participation/auction_used_allocations.proto

```diff
@@ -1,12 +1,17 @@
 syntax = "proto3";
 package tendermint.spn.participation;
 
+import "gogoproto/gogo.proto";
+
 option go_package = "github.com/tendermint/spn/x/participation/types";
 
 // Allocations used by a user for a specific auction
 message AuctionUsedAllocations {
   string address = 1;
   uint64 auctionID = 2;
-  uint64 numAllocations = 3;
-  bool withdrawn = 4;
+  bool withdrawn = 3;
+  string numAllocations = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
I suppose there's no running testnet for spn, because this change can break it :sweat_smile:

Reply:
No testnet

We also used to have a `alpha` and a `nightly` deployed instances so an instance stays sable